### ISSUE TITLE
Better distinction for System packages

### ISSUE THREAD

**Issue by psmulovics** [richorama/nuget-tree#11]
There are packages (like https://www.nuget.org/packages/System.Reactive/4.0.0-preview00001 ) which does start with "System." but are not part of the builtin/system dependencies.

**Comment by richorama** [comment]
Good point. Any idea where I can get a list of genuine 'system' packages?

**Comment by psmulovics** [comment]
hm, no better idea than asking the nuget team...
